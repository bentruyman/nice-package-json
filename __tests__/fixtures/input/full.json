{
  "private": "true",
  "engines": {
    "npm": "6.0.0",
    "node": "12.0.0"
  },
  "name": "full",
  "version": "1.2.3",
  "description": "Package description",
  "keywords": ["c", "b", "a"],
  "files": ["test", "src"],
  "bin": "bin.js",
  "main": "index.js",
  "repository": {
    "url": "git@host.com:test/repo.git",
    "type": "git"
  },
  "homepage": "https://host.com/test/repo",
  "bugs": "https://host.com/test/repo/issues",
  "contributors": [
    {
      "name": "John Appleseed",
      "email": "john.appleseed@apple.com"
    },
    {
      "name": "Jane Appleseed",
      "email": "jane.appleseed@apple.com"
    }
  ],
  "author": "Steve Wozniak <steve@woz.org>",
  "license": "MIT",
  "scripts": {
    "build": "./build.sh",
    "test": "./test.sh",
    "lint": "./lint.sh"
  },
  "bundledDependencies": {
    "lodash": "^2.0.0",
    "express": "^1.0.0"
  },
  "optionalDependencies": {
    "fibers": "^2.0.0",
    "bluebird": "^1.0.0"
  },
  "peerDependencies": {
    "vue-template-compiler": "^2.0.0",
    "vue": "^1.0.0"
  },
  "devDependencies": {
    "jest": "^2.0.0",
    "eslint": "^1.0.0"
  },
  "dependencies": {
    "lodash": "^2.0.0",
    "express": "^1.0.0"
  },
  "publishConfig": {
    "registry": "https://internal.host.com/registry",
    "access": "public"
  },
  "jest": {
    "bail": 1,
    "verbose": true
  },
  "eslintConfig": {
    "extends": "standard"
  }
}
