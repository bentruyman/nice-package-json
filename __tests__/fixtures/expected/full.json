{
  "name": "full",
  "private": "true",
  "version": "1.2.3",
  "description": "Package description",
  "keywords": [
    "a",
    "b",
    "c"
  ],
  "engines": {
    "node": "12.0.0",
    "npm": "6.0.0"
  },
  "main": "index.js",
  "bin": "bin.js",
  "files": [
    "src",
    "test"
  ],
  "homepage": "https://host.com/test/repo",
  "bugs": "https://host.com/test/repo/issues",
  "repository": {
    "type": "git",
    "url": "git@host.com:test/repo.git"
  },
  "author": "Steve Wozniak <steve@woz.org>",
  "contributors": [
    {
      "name": "Jane Appleseed",
      "email": "jane.appleseed@apple.com"
    },
    {
      "name": "John Appleseed",
      "email": "john.appleseed@apple.com"
    }
  ],
  "license": "MIT",
  "scripts": {
    "build": "./build.sh",
    "lint": "./lint.sh",
    "test": "./test.sh"
  },
  "dependencies": {
    "express": "^1.0.0",
    "lodash": "^2.0.0"
  },
  "devDependencies": {
    "eslint": "^1.0.0",
    "jest": "^2.0.0"
  },
  "bundledDependencies": {
    "express": "^1.0.0",
    "lodash": "^2.0.0"
  },
  "optionalDependencies": {
    "bluebird": "^1.0.0",
    "fibers": "^2.0.0"
  },
  "peerDependencies": {
    "vue": "^1.0.0",
    "vue-template-compiler": "^2.0.0"
  },
  "publishConfig": {
    "access": "public",
    "registry": "https://internal.host.com/registry"
  },
  "eslintConfig": {
    "extends": "standard"
  },
  "jest": {
    "bail": 1,
    "verbose": true
  }
}
